doc/rules/class_notation/final_internal_class.rst

```diff
@@ -40,6 +40,16 @@ Allowed types: ``bool``
 
 Default value: ``false``
 
+``attribute_exclude``
+~~~~~~~~~~~~~~~~~~~~~
+
+Class level attributes that must be omitted to fix the class, even if all of the
+white list ones are used as well. (case insensitive)
+
+Allowed types: ``array``
+
+Default value: ``['Entity', 'ORM\\Entity', 'ORM\\Mapping\\Entity', 'Mapping\\Entity']``
```

Comment:
Why is this list missing `Document` and `ODM\\Document` that are in `annotation_exclude`?

Reply:
The scope of scope the PR is the linked issue, there is no mention of `Document`. I didn't find it in the doctrine documentation either. We should keep PR's small, so I won't add features that were not requested, you can PR yourself if you want.

Reply:
The scope of this PR is attribute support, and support for mentioned classes is already [merged](https://github.com/doctrine/mongodb-odm/pull/2344) and will go live in next release, I don't understand why they can be added in this PR?

> We should keep PR's small

Absolutely, definitely, that's why I want https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5900 to be merged and improved later step-by-step and what's why I had objection with https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5934 which was too big to decent review as [few](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5962) places were missed.

Reply:
```suggestion
Default value: ``['Entity', 'ORM\\Entity', 'ORM\\Mapping\\Entity', 'Mapping\\Entity',m 'Document', 'ODM\\Document']``
```
as we use in other fixer - `final_internal_class`

Reply:
>The scope of this PR is attribute support, 

no, it is my PR and I set the scope, go do something else than spamming my PRs